storage/container_storage_interface/persistent-storage-csi-secrets-store.adoc

```diff
@@ -0,0 +1,50 @@
+[id="persistent-storage-csi-secrets-store"]
+= Secrets Store CSI Driver
+include::_attributes/common-attributes.adoc[]
+:context: persistent-storage-csi-secrets-store
+
+toc::[]
+
+[id="persistent-storage-csi-secrets-store-overview"]
+== Overview
```

Comment:
@lpettyjo Do you think you could move the contents of this section to a concept module? I might want to also reuse this in my assembly.

Reply:
Done.